cpp/log/etcd_consistent_store_test.cc

```diff
@@ -736,7 +735,7 @@ TEST_F(EtcdConsistentStoreTest, TestCleansUpToNewSTH) {
   {
     const StatusOr<int64_t> num_cleaned(CleanupOldEntries());
     ASSERT_OK(num_cleaned.status());
-    EXPECT_EQ(2, num_cleaned.ValueOrDie());
+    EXPECT_EQ(5, num_cleaned.ValueOrDie());
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7648032/7345053/46a92a08-ecc9-11e4-8096-ba36f0f19402.png)

We can't keep track of how many were deleted `OK`?